lib/tpm2_openssl.h

```diff
@@ -53,6 +57,15 @@
  */
 typedef unsigned char *(*digester)(const unsigned char *d, size_t n, unsigned char *md);
 
+/**
+ * Get an openssl hash algorithm ID from a tpm hashing algorithm ID.
+ * @param algorithm
+ *  The tpm algorithm to get the corresponding openssl version of.
+ * @return
+ *  The openssl hash algorithm id.
+ */
+UINT16 tpm2_openssl_halgid_from_tpmhalg(TPMI_ALG_HASH algorithm);
```

Comment:
is it really a uint16, usually openssl is int heavy

Reply:
Yes I think that's correct, the RSA_verify tool that this is used for is expecting an int.  This has been updated.